Resources/Locale/en-US/_Shitmed/autodoc.ftl

```diff
@@ -21,33 +21,51 @@ autodoc-error-hand-full = ITEM MANIPULATOR FULL
 autodoc-error-storage-full = ITEM TRAYS FULL
 autodoc-error-patient-unsedated = PATIENT REQUIRES SEDATION
 
-# These intentionally have russian alongside them for ostranauts-like aesthetic.
-# If you are a russian fork, these don't need translating :)
-# Если вы являетесь русскоязычным форком, то они не нуждаются в переводе :)
-autodoc-title = AUTODOC АВТОДОК MK.XIV
-autodoc-create-program = NEW PROGRAM НОВОЕ РАСПИСАНИЕ
-autodoc-program-title = PROGRAM TITLE ТИТУЛ РАСПИСАНИЯ
+# These intentionally have chinese alongside them for aesthetic purposes.
+# I barely speak chinese, but I double checked I wasn't saying a slur with google translate, so this should be ok. Especially since 99.9% of our players don't speak chinese anyway.
+
+# Chinese text translates approximately to "People's Glorious Automatic Surgery Machine"
+autodoc-title = AUTODOC 人民辉煌自动手术机 MK.XIV
```

Comment:
this feels more like a shitpost than actual flavouring of "this is a machine from (a stand-in for) communist china"

Reply:
yeah i had old broken mandarin that was more 1:1 that i can use if i find it